File: trunk/OscarDotNet/component/biz/Class_biz_officerships.pas
unit Class_biz_officerships;

interface

uses
  Class_db_officerships;

type
  TClass_biz_officerships = class
  private
    db_officerships: TClass_db_officerships;
  public
    constructor Create;
    procedure Bind
      (
      member_id: string;
      target: system.object
      );
  end;

implementation

constructor TClass_biz_officerships.Create;
begin
  inherited Create;
  // TODO: Add any constructor code here
  db_officerships := TClass_db_officerships.Create;
end;

procedure TClass_biz_officerships.Bind
  (
  member_id: string;
  target: system.object
  );
begin
  db_officerships.Bind(member_id,target);
end;

end.

File: component/biz/Class_biz_officerships.cs
using System;
using Class_db_officerships;
namespace Class_biz_officerships
{
    public class TClass_biz_officerships
    {
        private TClass_db_officerships db_officerships = null;
        //Constructor  Create()
        public TClass_biz_officerships() : base()
        {
            // TODO: Add any constructor code here
            db_officerships = new TClass_db_officerships();
        }
        public void Bind(string member_id, object target)
        {
            db_officerships.Bind(member_id, target);
        }

    } // end TClass_biz_officerships

}
